include/envoy/stats/BUILD

```diff
@@ -10,7 +10,19 @@ envoy_package()
 
 envoy_cc_library(
     name = "stats_interface",
-    hdrs = ["stats.h"],
+    hdrs = [
```

Comment:
can BUILD targets be split to multiple ones?

Reply:
Yes, but I was thinking of doing that in a follow-up as that will be another large PR.